pkg/manifest/anaconda_installer.go

```diff
@@ -370,7 +370,9 @@ func (p *AnacondaInstaller) liveStages() []*osbuild.Stage {
 	dracutOptions.AddDrivers = p.AdditionalDrivers
 	stages = append(stages, osbuild.NewDracutStage(dracutOptions))
 
-	stages = append(stages, osbuild.NewSELinuxConfigStage(&osbuild.SELinuxConfigStageOptions{State: osbuild.SELinuxStatePermissive}))
+	stages = append(stages, osbuild.NewSELinuxStage(&osbuild.SELinuxStageOptions{
+		FileContexts: "etc/selinux/targeted/contexts/files/file_contexts",
```

Comment:
I would suggest to rework the AnacondaInstaller pipeline a bit more, to be closer to how the `Os` pipeline handles SELinux context. The actual policy to be used is set in the pipeline options and then used when adding the stage 
https://github.com/osbuild/images/blob/0a9d89e4a15c98fca524591612fac8d90dc18943/pkg/manifest/os.go#L830

Consequently, it would be good to always make sure that the proper `selinux-policy-*` package is installed in the tree, similar to how it is done in https://github.com/osbuild/images/blob/0a9d89e4a15c98fca524591612fac8d90dc18943/pkg/manifest/os.go#L232-L234.

Reply:
Right I considered doing this the same as the OS pipeline but then felt that it didn't make much sense to configure this in the pipeline as it's non-changing and only used by a single image type.

Making sure `selinux-policy-x` is installed is something that should be done so that probably tips it the other way (make it configurable).

Reply:
New commit added that makes this now (similar to) the os pipeline; the value is hardcoded in the image but could be moved up even more later on if we wish (e.g. supporting `NoSELinux`).

It is a programming error to set a policy on the non-live installers.